app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player

```

Comment:
fix imports
![Screenshot from 2023-10-21 17-41-12](https://github.com/mate-academy/py-elves-and-dwarves/assets/122517203/41ca683b-3332-4238-9717-43d1abfb506f)